src/ts/ove.ts

```diff
@@ -125,6 +124,7 @@ export function displayPlasmidViewer(about: DOMStringMap): void {
         showMenuBar: false,
         withRotateCircularView: false,
         showReadOnly: false,
+        disableSetReadOnly: true,
```

Comment:
what is the purpose of this setting?

Reply:
The two settings will show "readonly" in the status bar and will prevent the users from changing it to "editable".
![20220626_170837.jpg](https://user-images.githubusercontent.com/65481677/175839415-815fb13d-3507-4d33-a5aa-0d59d74c4e39.jpg)

Reply:
From the ove demo page

pass showReadOnly=false to the <Editor> to not display the read-only <--> editable mode toggle, true by default

pass disableSetReadOnly=true to the <Editor> to not give users the option to change between read-only <--> editable mode, false by default